.github/workflows/manual.yml

```diff
@@ -0,0 +1,36 @@
+on: 
+  workflow_dispatch:
```

Comment:
umm if we trigger the action manually, it might not matter which repo it lives in?

Previously it was pointed out in review that we put this in engine, so that github event happens on engine and triggers workflow from engine.

Reply:
If that's the case, let's keep everything in Cocoon as its less infra intensive to run presubmits on

Reply:
Umm put a copy in Cocoon at https://github.com/flutter/cocoon/pull/3243.

https://github.com/flutter/cocoon/pull/3243#discussion_r1387283037 sir @godofredoc pointed out cocoon might not be an ideal place.

(Personally I am good with any repo since the workflow is triggered manually, but preferably living on engine since it might get triggered here in the future)